settei/parse_env.py

```diff
@@ -76,3 +78,95 @@ def parse_uuid(v: str) -> uuid.UUID:
 
     """
     return uuid.UUID(v)
+
+
+class EnvReader(typing.Mapping):
+    DELIMITER = '__'
+    ASTERISK_CHAR = 'ASTERISK'
+    LIST_CHAR = 'SETTEIENVLIST'
+
+    def __init__(
+        self,config: typing.Mapping[str, object] = {},
+        froms: typing.Optional[typing.Sequence[str]] = None, **kwargs
+    ):
+        self.config = dict(config, **kwargs)
+        self.froms = froms
+
+    def __iter__(self):
+        return self.config.__iter__()
+
+    def __len__(self):
+        return self.config.__len__()
+
+    def __getitem__(self, key: str) -> typing.Any:
+        result = None
+        try:
+            result = self.config[key]
+        except KeyError:
+            os_key = upper_key = key.upper()
+            if self.froms is not None:
+                os_key = self.DELIMITER.join([self.froms] + [upper_key])
```

Comment:
`froms`가 리스트가 아니라면 

```suggestion
                os_key = self.DELIMITER.join([self.froms, upper_key])
```

정도로 고칠 수 있을 거같네요

Reply:
위에 type annotation에서는 리스트여서 혹시 이거 `[[], item]`이런걸 의도한건가요?

Reply:
앗... 무슨 생각이었는지는 기억이 잘 안나지만 str이 맞는 것 같습니다!